src/demo/demo.stache

```diff
@@ -7,9 +7,9 @@
 		slides: slides,
 		
 		carouselOptions: { 
-			{{#each ./carouselOptions }} {{#is scope.key 'navArrows'}} 
+			{{#each(./carouselOptions)}} {{#is scope.key 'navArrows'}}
 			navArrows: { 
-				{{#each .}}
+				{{#each(.)}}
 				{{scope.key}}: '{{.}}', 
```

Comment:
indentation please

Reply:
unfortunately i can't clean this up because the output is in a `<pre>` tag which means if i properly indent the code, the output will be indented too much.